thunder/core/transforms.py

```diff
@@ -2299,6 +2312,19 @@ def iter_bound_symbols(bound_symbols):
             yield symbol
 
 
+def get_first_proxy(proxies) -> Proxy | None:
```

Comment:
So we need to merge https://github.com/Lightning-AI/lightning-thunder/pull/1755 first. I'll check if it's easy to pull it out from the current PR stack.